src/shared-components/BottomNavigation/index.tsx

```diff
@@ -0,0 +1,106 @@
+import HomeIcon from '@mui/icons-material/Home';
+import SyncAltOutlinedIcon from '@mui/icons-material/SyncAltOutlined';
+import DataSaverOffOutlinedIcon from '@mui/icons-material/DataSaverOffOutlined';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+import { styled } from '@mui/material/styles';
+import { Typography } from '@mui/material';
+import { useState } from 'react';
+import { colors } from 'src/styles/colors';
+
+const StyledDivBottomNavigation = styled(`div`)(({}: {}) => ({
+  display: 'flex',
+  width: '375px',
```

Comment:
Don't restrict the width, element should be as wide as the width of the application (and it's flexible)